### PR TITLE
12/grab codeblocks bugfix

### PR DESCRIPTION
**Pull request by david26694**
Solves https://github.com/koaning/mktestdocs/issues/12. 

It turns out that it is a bug indeed. Wrote unit test, added some formatting to docstrings before going to dedent.

### CODE REVIEW

File: tests/test_format_docstring.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+"""
+The docstring of the first class starts like
+\"""Some class ...
+The docstring of the second class starts like
+\"""
+Some class ...
+The tests are checking that regarddless of how the class docstring starts we should always be able to read the tests
+"""
```
Comment (koaning) [inline tests/test_format_docstring.py:32]: Shouldn't these lines be comments instead?

**Review by koaning — COMMENTED** [review]
Will need to figure out some time to do a proper deep dive later, but left a comment for now. Thanks!

File: tests/test_format_docstring.py (hunk 1 above)
Comment (david26694) [reply tests/test_format_docstring.py]: right, thanks!

File: tests/test_format_docstring.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    assert formatted == docstring
+
+
+
+# The docstring of the first class starts like
+# """Some class ...
+# The docstring of the second class starts like
+# """
+# Some class ...
+# The tests are checking that regarddless of how the class docstring starts we should always be able to read the tests
```
Comment (koaning) [inline tests/test_format_docstring.py:30]:
```suggestion
# The tests are checking that regardless of how the class docstring starts we should always be able to read the tests
```

File: tests/test_format_docstring.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    ```python
+    assert False, "this should fail."
+    ```
+    
+    """
+    def __init__(self):
+        pass
+
+
+bad_members = get_codeblock_members(BadClass)
```
Comment (koaning) [inline tests/test_format_docstring.py:55]:
```suggestion
```

Comment (koaning) [reply tests/test_format_docstring.py:55]: This is not used anywhere, or am I mistaken?

**Comment by koaning** [comment]
Figured I might as well make the changes manually so that I might make a release. Less back and forth that way.

**Pull request merged**